_data/communication.yml

```diff
@@ -162,7 +162,8 @@ websites:
       sms: Yes
       software: Yes
       email: Yes
-      doc: https://help.salesforce.com/HTViewHelpDoc?id=security_require_two_factor_authentication.htm
+      hardware: Yes
```

Comment:
wait a sec, why did you add hardware? we consider mobile authenticator apps to still be "software".
"hardware" are physical fobs or yubikey and similar type devices

Reply:
Yup I saw on that admin page that they supported U2F :)